CMakeLists.txt

```diff
@@ -27,6 +27,18 @@ list( APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
 project( nest CXX C )
 set( NEST_USER_EMAIL "users@nest-simulator.org" )
 
+# this include for coloring messages
+include( Utils )
+# check if the given CMAKE_INSTALL_PREFIX is not empty
+if( NOT IS_ABSOLUTE ${CMAKE_INSTALL_PREFIX})
```

Comment:
Following the *principle of least surprise*, I think we should just convert a non-absolute path to an absolute one (see [stackoverflow/39027269](https://stackoverflow.com/questions/39027269)).